plugins/modules/cloud/linode/linode_v4.py

```diff
@@ -57,6 +57,12 @@
          method for marking instances is to use tags.
     type: str
     required: false
+  private_ip:
+    description:
+      - If true, the created Linode will have private networking enabled and
+        assigned a private IPv4 address.
+    type: bool
+    default: false
```

Comment:
Is this an option which is only used during creation of instances, or also for idempotency checks? If the latter, there should not be a default, so in the code one can distinguish this option not being specified from this option being explicitly set to `false`.

Reply:
This is only used during instance creation. Idempotency is checked by instance label.